tests/unit-tests/tools/OpenDDS_RtpsRelayLib/Name.cpp

```diff
@@ -1,10 +1,10 @@
 #ifdef OPENDDS_HAS_CXX11
 
-#include <gtest/gtest.h>
+#include <OpenDDS_RtpsRelayLib/Name.h>
```

Comment:
We don't have this style of directory name anywhere else in the repo.  Since `$(DDS_ROOT)/tools` is on the include path, how about making this include `dds/rtpsrelaylib/Name.h` with the actual file being `tools/dds/rtpsrelaylib/Name.h`?  Then we have the same top-level directory after `$prefix/include` as the one that's already in use.

Reply:
> We don't have this style of directory name anywhere else in the repo.

No, I based it on the library filenames but I'd like us to eventually transition to something where the include path starts with some form of OpenDDS. This is related to https://github.com/objectcomputing/OpenDDS/discussions/2160.

>  Since $(DDS_ROOT)/tools is on the include path, how about making this include dds/rtpsrelaylib/Name.h with the actual file being tools/dds/rtpsrelaylib/Name.h?

Maybe... but I don't know about using `dds` because of the reason above and I'm not sure it should be the same name as the main directory that it's not part of. It sounds a bit confusing.

Reply:
Here's an example from ACE https://github.com/DOCGroup/ACE_TAO/tree/master/ACE/protocols/ace
We can certainly eventually transition to something else, but I'd rather not do that for different libraries at different times

Reply:
The fact that ACE does it doesn't make me feel any better about it.

Reply:
Yes, I was assuming that was the case when I wrote the last comment.
However I do think we need to be consistent across different libraries in the same version.  A hypothetical change to a new scheme should come later and apply to all libraries.